adjustment for the input file does not overflow in Firefox

teamcreed opened the pull request:
adjustment for the input file does not overflow in Firefox

teamcreed committed:
Update bootstrap.css

adjustment for the input file does not overflow in Firefox

teamcreed committed:
Update forms.less

adjustment for the input file does not overflow in Firefox

cvrebert commented:
Could you post an example that demonstrates the problem which your change fixes?

cvrebert commented on less/forms.less:
```diff
@@ -138,6 +138,10 @@ output {
 
+//adjustment for the input file does not overflow in Firefox 
+&[type=file]{
+  height:auto;
```
Missing a space after the colon

cvrebert commented on less/forms.less:
```diff
@@ -138,6 +138,10 @@ output {
   }
 
+//adjustment for the input file does not overflow in Firefox 
+&[type=file]{
```
Missing quotation marks around `file`

vanckruz committed:
creating a lateral menu , admin style - Jhonny Vasquez

teamcreed committed:
Update navs.less

tabs sidebar - Jhonny Vasquez

teamcreed committed:
Update forms.less

adjustment for the input file does not overflow in Firefox - adding cuotes in "file" and blank space after the colon and blank space after the colon.

Example :

cvrebert commented:
Closing per https://github.com/twbs/bootstrap/issues/18025#issuecomment-150654830

cvrebert commented:
With regard to the unrelated sidebar nav feature, all proposed new features should target Bootstrap v4 (the `v4-dev` branch).
Bootstrap v3 (the `master` branch) is in maintenance mode and is no longer accepting new features.
